### PR TITLE
Ele 227 configuring a timestamp column that

### PR DESCRIPTION
**Pull request by Maayan-s**
Fixes https://github.com/elementary-data/elementary/issues/610

### CODE REVIEW

File: macros/edr/tests/test_utils/find_normalized_data_type_for_column.sql [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {% set columns_from_relation = adapter.get_columns_in_relation(model_relation) %}
 {% if columns_from_relation and columns_from_relation is iterable %}
     {% for column_obj in columns_from_relation %}
-        {% if column_obj.column | lower == column_name | lower %}
+        {% if column_obj.column | lower == column_name | lower or column_obj.column | lower == column_name[1:-1] | lower %}
```
Comment (RoiTabach) [inline macros/edr/tests/test_utils/find_normalized_data_type_for_column.sql:6]: This is python, right? 
how about 
```suggestion
        {% if column_obj.column | lower == column_name | lower or column_obj.column | lower == column_name.replace('"','') | lower %}
```

Comment (Maayan-s) [reply macros/edr/tests/test_utils/find_normalized_data_type_for_column.sql]: Made a change to be more explicit, thanks

File: macros/edr/tests/test_utils/find_normalized_data_type_for_column.sql [hunk 2]
```diff
@@ -3,7 +3,7 @@
 {% set columns_from_relation = adapter.get_columns_in_relation(model_relation) %}
 {% if columns_from_relation and columns_from_relation is iterable %}
     {% for column_obj in columns_from_relation %}
-        {% if column_obj.column | lower == column_name | lower %}
+        {% if column_obj.column | lower == column_name | trim('\"') | trim('\'') | trim('\`') | lower %}
```
Comment (elongl) [inline macros/edr/tests/test_utils/find_normalized_data_type_for_column.sql:6]:
```suggestion
        {% if column_obj.column | lower == column_name | trim('\'\"\`') | lower %}
```

**Comment by elongl** [comment]
Have we tested that the rest of the logic works as expected?

**Comment by Maayan-s** [comment]
@elongl 
I checked locally with a column as the user described and it worked. It still leaves the responsibility on the user to pass the timestamp column quoted.
But they do the same with dbt source freshness, so I think it's valid. 
The only place where we failed because of the quotes was here.

**Pull request merged**